### PR TITLE
Main UI activity and empty mvp architecture created

### PR DESCRIPTION
**Pull request by Faka-GLB**
Main UI for the app and its corresponding mvp classes created, now empty.
Renamed app to PronosticAR.

image

PronosticAR project tree

### CODE REVIEW

File: app/src/main/java/com/onboarding/pronosticAR/mvp/view/base/FragmentView.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+open class FragmentView(fragment: Fragment) {
+    private val fragmentRef: WeakReference<Fragment> = WeakReference<Fragment>(fragment)
+
+    val fragment: Fragment?
+        get() = fragmentRef.get()
+
+    val context: Context?
+        get() = fragment?.context
+}
```
Comment (mauroLonghi) [inline app/src/main/java/com/onboarding/pronosticAR/mvp/view/base/FragmentView.kt]: mmm you not need a fragment now, remove this

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -7,4 +7,5 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="main_text_color">#FFA617</color>
```
Comment (mauroLonghi) [inline app/src/main/res/values/colors.xml]: pls align with the correct code style

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="text_size_title">30sp</dimen>
+    <dimen name="text_size_normal">20sp</dimen>
+    <dimen name="main_activity_title_margin_xsmall">10dp</dimen>
```
Comment (mauroLonghi) [inline app/src/main/res/values/dimens.xml]: code style its missing

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="TitleTextStyle" parent="TextAppearance.AppCompat.Display1">
```
Comment (mauroLonghi) [inline app/src/main/res/values/styles.xml]: not represent the WHERE in the code style

**Comment by mauroLonghi** [comment]
Also, move the image in drawable-24 to the package drawable (and rename plase)

File: app/src/main/java/com/onboarding/pronosticAR/mvp/view/base/FragmentView.kt (hunk 1 above)
Comment (Faka-GLB) [reply app/src/main/java/com/onboarding/pronosticAR/mvp/view/base/FragmentView.kt]: removed

File: app/src/main/res/values/colors.xml (hunk 2 above)
Comment (Faka-GLB) [reply app/src/main/res/values/colors.xml]: done

File: app/src/main/res/values/dimens.xml (hunk 3 above)
Comment (Faka-GLB) [reply app/src/main/res/values/dimens.xml]: done

File: app/src/main/res/values/styles.xml (hunk 4 above)
Comment (Faka-GLB) [reply app/src/main/res/values/styles.xml]: done

**Review by mauroLonghi — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Review by BQuinteros — APPROVED** [review]

**Pull request merged**